lang/src/org/partiql/lang/syntax/SqlParser.kt

```diff
@@ -573,7 +573,8 @@ class SqlParser(
                         groupBy_(
```

Comment:
The change in this file were needed to add some missing `SourceLocationMeta`s that were missing from a few different nodes.  We need those to provide line/col info in our error messaging.